src/mosquito/redis_backend.cr

```diff
@@ -159,7 +159,7 @@ module Mosquito
       time = Time.utc
       overdue_job_runs = redis.zrangebyscore(scheduled_q, "0", time.to_unix_ms.to_s).as(Array)
 
-      return [] of JobRun unless overdue_job_runs.any?
+      return [] of JobRun if overdue_job_runs.empty?
```

Comment:
Normally, I'd ask for ameba fixes to be submitted separately, just so the changes history doesn't get too muddled. In this case, 👍 and thank you.